goalgorilla/open_social
# Issue #3218654 by nkoporec: Display the correct event type when displaying event teasers

## Problem
Currently when using the social blue theme with the new style (Sky) set, the event teasers are displaying 'event' text above the event title. When using the social_event_type module we can distinguish between different types, so it would make a lot more sense to display the actual event type above the title instead of the generic "Event" text.

## Solution
We need to pass the event type to node teaser rendering template.

## Issue tracker
https://www.drupal.org/project/social/issues/3218654

## How to test
1. Set social blue theme
2. Under theme configuration, set the style setting to Sky (New)
3. Enable the social_event_type module
4. Create new event types and create a new event
5. Go to /community-events page and see the teasers, even though we have different type set on an event, the text above the event title still displays "Event".

## Screenshots
Before:
![Screenshot from 2021-06-14 11-34-23](https://user-images.githubusercontent.com/35064680/121871601-99460c00-cd04-11eb-919f-cbefc7b7772a.png)

After:
![Screenshot from 2021-06-14 11-35-00](https://user-images.githubusercontent.com/35064680/121871641-a19e4700-cd04-11eb-8007-527490e45d9c.png)


## Release notes

When using social_event_type module, the event teasers will now display actual event type above the event title.

## Change Record
/

## Translations
/
